src/content/en/2020/third-parties.md

```diff
@@ -68,76 +72,95 @@ _Note on CDNs: The CDN category here includes providers that provide resources o
+* Only the home pages are tested. Other pages may having difference third-party requirements.
 * Roughly 84% of all third-party domains by request volume have been identified and categorized. The remaining 16% fall into the "Other" category.
 
+Learn more about our [methodology](./methodology).
+
 ## Prevalence
 
 A good starting point for this analysis is to confirm the statement that third-party content is a critical component of most websites today. How many websites use third-party tags, and how many tags do they use?
 
+{# TODO Authors - is "tags" the right wording here? Should we just use "content"? #}
+
 {{ figure_markup(
   image="pages-with-thirdparties.png",
   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSNXusoFJKi8Z1i_yuG_5umUCJOZwBMaZaECq4T8lU5zIQuLR7UHfZOJmXsXzPWQTJFnFa3dcOEPJgy/pubchart?oid=1149547895&format=interactive",
-  caption="Third party content prevalence has grown slightly since 2019",
-  description="In 2019 93.6% of mobile pages had third party content, in 2020 this was 94.1%. In 2019 93.6% of desktop pages had third party content, in 2020 this was 93.9%.",
+  caption="Pages with third-party content",
+  description="Third party content prevalence has grown slightly since 2019. In 2019 93.6% of mobile pages had third-party content, in 2020 this was 94.1%. In 2019 93.6% of desktop pages had third-party content, in 2020 this was 93.9%.",
   width=600,
-  height=371
+  height=371,
+  sheets_gid="1477664642",
+  sql_file="percent_of_websites_with_third_party.sql"
   )
 }}
 
+{# TOD (authors and analysts) - I don't see 2019 data in the SQL query. Did you get that from last year's chapter (in which case we should mark it as such in the Spreadsheet) or did you run a different SQL query (in whcih case we should update it) #}
+
 These prevalence numbers show a slight increase on [the 2019 results](../2019/third-parties): 93.87% of pages in the desktop crawl had at least one third-party request, the number was slightly higher at 94.10% of pages in the mobile crawl. A brief look into the small number of pages with no third-party content revealed that many were adult sites, some were government domains and some were basic landing / holding pages with little content. It is fair to say that the vast majority of pages have at least one third-party.
 
+{# TODO authors - that's interesting that it's still increasing. I would have thought that potentially the prevalance of cookie banners preventing some third-party content loading by default might cause a slight decrease this year. Though I guess that's only few countries like Europe, and mostly concerned with Advertising/Tracking third-party content. Also cookie trackers themsleves are often implemented as third-party CMP. Still interesting none-the-less. Is it worth noting this? Maybe with link to privacy chapter. #}
```

Comment:
@simonhearne WDYT? I see you did cover this down below, and in conclusion, but still wonder if worth commenting on this here?